rrfs-test/CMakeLists.txt

```diff
@@ -22,22 +38,10 @@ if (CLONE_RRFSDATA)
          set(MPI_ARGS "${MPI_ARGS} --exclusive")
       endif()   
 
-      # JEDI test cases
-      # ----------------
-      # NOTE: To add a new ctest for FV3, simple add the new yaml file here (without the extension)
-      #       Also need to add the associated executable to the rrfs_fv3jedi_exes list below 
-      list( APPEND rrfs_fv3jedi_tests
-             rrfs_fv3jedi_hyb_2022052619
-             rrfs_fv3jedi_letkf_2022052619
-          )
-
-      # JEDI executables for each test case above
-      list ( APPEND rrfs_fv3jedi_exes
-             fv3jedi_var.x
-             fv3jedi_letkf.x
-      )
-
-      foreach(case IN LISTS rrfs_fv3jedi_tests)
+      foreach(keypair ${rrfs_fv3jedi_tests})
```

Comment:
#dic1
Then, in this part, all are the same as the version prior to this PR.  Only need to change the line 44 as
```
set (exe $get_from_dictionary(rrfs_tests_execs($case)) ! the exact syntax needs check. 

```
Will this way be good to make use of the dictionary-like structure you added?